ethnodes/ethnodes-dockerfile

```diff
@@ -0,0 +1,77 @@
+FROM debian:jessie
```

Comment:
I don't really know docker things - what does this do?

Reply:
This builds the MySQL 5.7 image based on Debian Jessie image. Near the end, the `entrypoint.sh` is copied to the image and set as the default entrypoint of the container.
This is also on the official mysql container build script. I added the source and changes I made at the top of this file.